tests/integration/utils.py

```diff
@@ -93,6 +94,42 @@ def wait_for_complete_request(self, response):
 
         return response
 
+    def fetch_all_requests(self, page=None, per_page=None, state=None, verbose=None):
+        """
+        Fetch a list of requests from the Cachito API.
+
+        :return: Object that contains response from the Cachito API
+        :rtype: list
+        """
+        page_tuple = ("page", page)
+        per_page_tuple = ("per_page", per_page)
+        state_tuple = ("state", state)
+        verbose_tuple = ("verbose", verbose)
+        url_values = urllib.parse.urlencode(
+            {
+                k: v
+                for k, v in [page_tuple, per_page_tuple, state_tuple, verbose_tuple]
+                if v is not None
+            }
+        )
+        result_url = f"{self._cachito_api_url}/requests?{url_values}"
+        resp = requests.get(result_url)
+        resp.raise_for_status()
+
+        if all(parameter is None for parameter in [page, per_page]):
```

Comment:
`per_page` is never `None`. This code seems not needed.

Reply:
Just used `... request_json={}):`

Reply:
Sure, but all the calls to `fetch_all_requests` set `per_page` in `request_json` to something other than `None`. This still looks like dead code to me.

Reply:
I check now that `if not page`.